libkpod/stop.go

```diff
@@ -11,18 +11,22 @@ func (c *ContainerServer) ContainerStop(container string, timeout int64) (string
 	if err != nil {
 		return "", errors.Wrapf(err, "failed to find container %s", container)
 	}
+	ctrID := ctr.ID()
```

Comment:
Why is the ID being saved here?

Reply:
Just so to clean up the 4 or 5 ctr.ID() calls below and just do it once.

Reply:
Alright, sounds fine. Was just wondering if we were expecting the container to change out from under us or something similar.